deploy/roles/nodejs/tasks/main.yml

```diff
@@ -8,6 +8,32 @@
     - https://dl.yarnpkg.com/debian/pubkey.gpg
     - https://deb.nodesource.com/gpgkey/nodesource.gpg.key
 
+- name: Check if Node exists
```

Comment:
This is the part that automatically removes any Node version that is not version 18.

Reply:
I should have been clearer here. Lines 11-45 take care of the Node upgrade.

Reply:
Ah, so this is not about the Vagrant environment. Is it right to say that these new tasks are intended to support transitioning the production, staging, and "sandbox" environments?

Reply:
Yes, that's right!

Reply:
Got it! How about placing instructions in a dedicated file and only including it for production? That will more accurately reflect the intent, optimize the "nodejs" role, and make it easier to safely remove the transitional tasks once all the running systems have been updated.

Reply:
@jugglinmike I placed the tasks [here](https://github.com/w3c/aria-at-app/pull/492/files#diff-2d12b0a825499de9d3504b2450eb0896dde07fb5374db0e5c5bf39b7870ae63d).